src/assets/scss/offerings.scss

```diff
@@ -0,0 +1,16 @@
+@import "colours";
+
+.card {
```

Comment:
Additionally, the card header is a different colour. I don't know if that was intentional or not.